uint8 nodeID
uint8 HopsUntilGateway 
uint8 PreferredNode
uint32 totalMessages

